opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -55,6 +55,9 @@ message ResourceLogs {
   // A list of ScopeLogs that originate from a resource.
   repeated ScopeLogs scope_logs = 2;
 
+  // The Schema URL, if known. This is the identifier of the Schema that the resource data
+  // is recorded in. To learn more about Schema URL see
+  // https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/schemas#schema-url
```

Comment:
Could we point this at the opentelemetry.io website?

I think we have more control over redirects there in the event we refactor the specification.

Reply:
I can't find the schema pages anywhere at https://opentelemetry.io/docs/specs/otel/

Looks like they are not published.

Reply:
Nevermind, it is there https://opentelemetry.io/docs/specs/otel/schemas/#schema-url

Reply:
Links now point to opentelemetry.io website.